ajax/libs/jquery.gray/package.json

```diff
@@ -4,16 +4,28 @@
   "version": "1.4.4",
   "description": "Make an image gray in all browsers",
   "license": "MIT",
-  "author": "kununu",
+  "author": "Karl Horky <karl.horky@gmail.com>",
   "keywords": [
     "gray",
     "grey",
     "grayscale",
     "images",
-    "image"
+    "image",
+    "jquery",
+    "jquery-plugin"
   ],
   "repository": {
     "type": "git",
     "url": "git://github.com/karlhorky/gray.git"
-  }
+  },
+  "npmName": "jquery-gray",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "css/*",
+        "js/*"
+      ]
+    }
+  ]
```

Comment:
Is there any reason that why to use npm auto-update?
how about use GitHub repository?
There are more versions, and also has the new version.